concent_api/core/views.py

```diff
@@ -0,0 +1,223 @@
+import json
+import datetime
+
+from django.http                    import HttpResponse
+from django.views.decorators.http   import require_POST
+from django.utils                   import timezone
+from django.conf                    import settings
+
+from utils.api_view                 import api_view, Http400
+from .models                        import Message, MessageStatus
+
+
+@api_view
+@require_POST
+def send(request, message):
+    data = message
```

Comment:
Przydałyby się komentarze do funkcji widoków.

Reply:
They are documented on the API level in Sphinx docs. This is my task though. I have these docs in a separate branch, waiting for this one to be merged first.

Also, documenting it in this particular branch would be counter-productive since it was just a temporary solution while we were waiting for golem-messages to be finalized.

But, just like I said about docstrings for models - adding more docs is a good idea as long as we're not spending all available time writing docs :) Any help in that regard will be appreciated.

Reply:
:+1: